### PR TITLE
show scope in editor element

### PR DESCRIPTION
**Pull request by weiner**
This PR adds the display of the scope to the EditorElement. This helps with orientation which property is represented by the UI element.

### CODE REVIEW

**Comment by eneufeld** [comment]
@weiner thank you for the contribution, could you clean up the contribution, there is something weird with your rebase?

File: jsonforms-editor/src/editor/components/EditorElement.tsx [hunk 1]
```diff
@@ -138,6 +144,20 @@ export const EditorElement: React.FC<EditorElementProps> = ({
               >{`(${ruleEffect})`}</Typography>
             </Grid>
           ) : null}
+          {scope ? (
```
Comment (eneufeld) [inline jsonforms-editor/src/editor/components/EditorElement.tsx:147]: Wouldn't it make sense to check for `isEditorControl(wrappedElement)` instead for the scope?
You could also do something like this:
`isEditorControl(wrappedElement) && (<Grid...>) `
this way you don't need an else case

**Review by eneufeld — CHANGES_REQUESTED** [review]
This looks good. Thank you for this.
The cypress tests are failing though:
- dnd.spec.js
  - Dnd Move Tests on Example model -- Move element in the same parent, to the right
- editor.spec.js
  - Remove controls -- Can remove control
  - Remove controls -- Can remove layout
These test select the grid index based, so they must be adapted.
Could you please fix them?

File: jsonforms-editor/src/editor/components/EditorElement.tsx (hunk 1 above)
Comment (weiner) [reply jsonforms-editor/src/editor/components/EditorElement.tsx]: I updated the test and inlined the check. 
Thanks for reviewing this.

What is your timeline on publishing the lib to npm?

Comment (eneufeld) [reply jsonforms-editor/src/editor/components/EditorElement.tsx]: Thank you for the fixes. It looks good now.
We plan to publish this library in Q1/21.

**Review by eneufeld — APPROVED** [review]
Thank you

**Pull request merged**